src/pages/Settings/TwoStepFaq.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+	<div>
+		<kv-settings-card>
+			<template v-slot:content>
+				<div class="strong">
+					Level up your security with 2-step verfication
+				</div>
+				<p>
+					2-step verification protects your Kiva account by requiring an
+					additional code when you log in on a device we don't recognize.
+				</p>
+				<div class="strong">
+					FAQ
+				</div>
+				<div>
+					<a @click="triggerWhatIsLightbox">What is 2-step Verification?</a>
```

Comment:
Since these do not change URLs, they should be set as button elements, rather than anchors.
You can match the styling using `<kv-button class="text-link" @click="...`